src/cf.data.pre

```diff
@@ -1645,6 +1644,22 @@ ENDIF
 		acl fileupload req_mime_type -i ^multipart/form-data$
 		acl javascript rep_mime_type -i ^application/x-javascript$
 
+
+	Built-in ACLs
+
+	The names of the following built-in ACLs can be used just like the names
+	of configurable ACLs (i.e. ACLs defined using the "acl" directive). These
+	built-in ACLs do not have reusable ACL types and cannot be expressed or
+	adjusted using "acl" directives. Configurable ACLs cannot be named using
+	the names of these built-in ACLs.
+
+	manager
+		Matches cache manager requests targeting this Squid instance. [fast]
+		Squid does not forward matching requests and, if possible, satisfies
+		them by responding with the requested cache manager report. Currently,
```

Comment:
I failed to come up with realistic/legitimate use cases where `manager` ACL matching inconsistent with routing decision may lead to problems (that Squid should be responsible for). That failure led to a better (IMO) way of looking at the problem itself: The `manager` ACL is not trying to guarantee that matching transactions (and only them) will be routed to cache manager rather than forwarded.  We are not developing a new `manager_access` directive here! A (mis)matching `manager` ACL guarantees nothing in general.

The guarantee we are providing is very specific to the moment(s) when Squid makes request routing decisions. In such a moment, a matching request (and only matching requests) will be routed to cache manager (rather than forwarded). That is all. Whether the same ACL matched or mismatched before that decision (e.g., during proxy_protocol_access checks) or after it (e.g., during http_reply_access checks) is irrelevant to Squid. The admin may use this ACL in any context with an HTTP request, and the admin is responsible for using it correctly.

The following (unformatted) change is meant to codify the above idea. If you can find use cases where the existing implementation will violate the documented guarantee, let's discuss. If you think we can provide a stronger guarantee, please let me know. Otherwise, please format and update `manager` documentation accordingly.

```suggestion
		When deciding how to route a transaction, Squid does not forward matching
		requests and, if possible, satisfies them by responding with the requested
		cache manager report. This ACL is typically used in http_access rules, but it
		may be used in any context with access to an HTTP request. Squid routing
		decisions do not depend on whether this ACL has been used or evaluated. Currently,
```

Reply:
> Squid routing decisions do not depend on whether this ACL has been used

What if we use this ACL in, e.g., `never_direct` directive to enforce a 'to-peer/to-origin' decision? Does this count as a 'routing decision' affected by this ACL?

Reply:
> > Squid routing decisions do not depend on whether this ACL has been used

> What if we use this ACL in, e.g., `never_direct` directive to enforce a 'to-peer/to-origin' decision? Does this count as a 'routing decision' affected by this ACL?

I should have said "These routing decisions" to be more precise, but more work is needed to make this text clear and correct. FWIW, by that "do not depend" phrase, I was trying to document that the following (and similar) possible designs are _not_ used by Squid:

* If no squid.conf directive uses a "manager" ACL, then all cache manager requests are automatically denied.
* If no "manager" ACL is evaluated when handling a cache manager request, then that cache manager requests is automatically denied.

I think we need to rework this documentation further, to avoid the (false or misleading) implication that cache manager requests are those that match the manager ACL. We want to say that internal cache manager requests will match this ACL, but whether this ACL is mentioned or actually matches in some directive, has no affect on:

* Whether internal cache manager requests are forwarded. They never are.
* Whether internal cache manager requests are routed to the cache manager module of the receiving Squid instance. They always are unless they are blocked on the way (by http_access denials or other obstacles).

We can (and should) polish this text further as long as we agree on the idea/principle itself.

Reply:
>  this ACL is mentioned or actually matches in some directive, has no affect on

Agreed, we should clarify this.